test/Hyperfund.t.sol

```diff
@@ -283,16 +291,17 @@ contract HyperfundTest is Test {
         emit Hyperfund.FractionRedeemed(fractionHypercertId + 1, token, _unitsToTokenAmount(multiplier, units));
         hyperfund.redeem(fractionHypercertId + 1, token);
         vm.stopPrank();
-        assertEq(hypercertMinter.ownerOf(fractionHypercertId + 1), contributor);
+        assertEq(hypercertMinter.ownerOf(fractionHypercertId + 1), address(0));
```

Comment:
but it should belong to the contributor

Reply:
nope, its burnt so zero address

Reply:
strange, the code in their repo doesn't do that, only zeros the units